### PR TITLE
Add elementId to R template function

### PR DESCRIPTION
**Pull request by timelyportfolio**
Regarding #208, add `elementId = NULL` to the `R` template function for `scaffoldWidget`.

### CODE REVIEW

File: inst/templates/widget_r.txt [hunk 1]
```diff
@@ -5,7 +5,10 @@
 #' @import htmlwidgets
 #'
 #' @export
-%s <- function(message, width = NULL, height = NULL) {
+%s <- function(message,
+  width = NULL, height = NULL,
+  elementId = NULL
+) {
```
Comment (ramnathv) [inline inst/templates/widget_r.txt]: Can you move all the function arguments to the same line?

Comment (timelyportfolio) [inline inst/templates/widget_r.txt]: done :)

**Comment by ramnathv** [comment]
I am good with this. @jcheng5 @jjallaire @yihui any objections to including this in the scaffolding template?

**Comment by timelyportfolio** [comment]
Can we push this through?

**Pull request merged**